.github/actions/cache-asdf/action.yml

```diff
@@ -0,0 +1,49 @@
+name: "Cache asdf installation"
```

Comment:
This file is copied over from the private repo. We could move it into its own repo and publish the action.

I also think it's worth considering removing `asdf` from CI because this isn't a common setup for JS projects. That's not a bad thing necessarily, but I think it's worth reconsidering (for CI specifically).